src/components/Settings/Export/Export.helpers.js

```diff
@@ -491,6 +508,10 @@ const addTileToGrid = async (
     alignment: 'center',
     width: '100'
   };
+  if (picsee) {
+    // scale down images to fit inside PicseePal dimensions
+    imageData.width = '60';
+  }
```

Comment:
You will need to add diferents `imageData.width` depending on the rows and columns of the board. This is made in line [500](https://github.com/cboard-org/cboard/pull/1302/files#diff-a5187bfb1603e0959ee1c612e12529bf7f9b80d2b8a56d5e8f7a8b390e9d058bR521) .  You can export a fixed board of 5 columns and 5 rows and will see  the last row is in other page. If the `imageData.width = '47' ` is exported correctly.

Reply:
I have implemented dynamic resizing of images based on the number of columns, in my newest commit. I have also tested PicseePal PDF exporting with all column sizes (1 through 12) to ensure nothing is cut off.